docs/QUICKSTART.md

```diff
@@ -1,21 +1,44 @@
 # Quickstart #
 
-The CLI requires a few dependencies and C extensions that can be installed with
-`pip install securesystemslib[crypto,pynacl]`.
+In this quickstart tutorial, we'll use the basic TUF command-line interface
+(CLI), which includes the `repo.py` script and the `client.py` script, to set
+up a repository with an update and metadata about that update, then download
+and verify that update as a client.
+
+Unlike the underlying TUF modules that the CLI uses, the CLI itself is a bit
+bare-bones.  Using the CLI is the easiest way to familiarize yourself with
+how TUF works, however.  It will serve as a very basic update system and use
```

Comment:
...? I think this is missing the end of the sentence.

Reply:
Thanks. ^_^